src/opamp.rs

```diff
@@ -911,6 +912,67 @@ opamps! {
     },
 }
 
+#[cfg(any(feature = "stm32g471", feature = "stm32g491", feature = "stm32g4a1"))]
+opamps! {
+    Opamp1 => opamp1: {
+        vinm0: crate::gpio::gpioa::PA3<crate::gpio::Analog>,
```

Comment:
You could reexport all pins from `gpio::gpiox` to `gpio` and get access by just `gpio::PA3`.